Makefile

```
@@ -5,4 +5,4 @@ install:
 	docker-compose run --rm test composer install
 
 update:
-	docker-compose run --rm test composer update
+	docker-compose run --rm test php -version && composer update && composer show -D > versions.txt
```

I don't think the `&&` works unless you use `bash -c "stuff"`:
```suggestion
	docker-compose run --rm test bash -c "php -version && composer update && composer show -D > versions.txt"
```

---

I went ahead and made the change, but it seems to run the same on my machine either way.